src/main/scala/gitbucket/core/util/RepoBase.scala

```diff
@@ -0,0 +1,3 @@
+package gitbucket.core.util
+
+case class RepoBase(baseUrl:String, sshAddress:Option[SshAddress])
```

Comment:
`Context` has both of `baseUrl` and `settings.sshAddress`. Is this case class necessary? It's possible to create repository urls from only `Context`, and `Context` is also available at places where `repoBase` is used in this pull request.